Cargo.toml

```
@@ -5,4 +5,4 @@ members = [
     "metriken",
     "metriken-core",
     "metriken-derive"
-]
+, "metriken-exposition"]
```

The formatting here is a little weird
```suggestion
    "metriken-derive",
    "metriken-exposition",
]
```

---

That is. Good to know that cargo new will mangle that.